api-specificatie/openapi.yaml

```diff
@@ -366,9 +372,9 @@ components:
       schema:
         type: string
     #JBO: Advies om hier een betekenisvollere naam te geven. Als je meerdere raadpleeg endpoints in 1 API wilt stoppen is dit een onduidelijke naamgeving. Voorstel: adresIdentificatie.
-    key:
-      description: key verkregen uit zoeksuggestie om meer informatie over dat resultaat op te vragen
-      name: key
+    nummeraanduidingidentificatie:
```

Comment:
Ik zou hier het volgende van maken:
    nummeraanduidingIdentificatie:
      description: Identificatie van een nummeraanduiding uit de BAG. Deze is 16 cijfers lang. 
      name: nummeraanduidingIdentificatie
      in: path
      required: true
      schema:
        type: string
        pattern: '^[0-9]{4}20[0-9]{10}$'
        example: 1234200000000001

Reply:
Ik heb wel quotes om de example gezet omdat dit element als string gedefinieerd is